docs/admin/logging/logentries.md

```diff
@@ -0,0 +1,42 @@
+<!--[metadata]>
++++
+aliases = ["/engine/reference/logging/logentries/"]
+title = "Logentries logging driver"
+description = "Describes how to use the Logentries logging driver."
+keywords = ["logentries, docker, logging, driver"]
+[menu.main]
+parent = "smn_logging"
+weight = 2
```

Comment:
And this as well; that way it'll sort just alphabetically in the menu (but let's keep that change for later as well, during docs review)